pom.xml

```diff
@@ -60,7 +60,7 @@
 
         <version.io.opentracing>0.31.0</version.io.opentracing>
         <version.org.awaitility>3.0.0</version.org.awaitility>
-        <version.org.springframework.boot>1.5.8.RELEASE</version.org.springframework.boot>
+        <version.org.springframework.boot>2.1.3.RELEASE</version.org.springframework.boot>
         <version.org.springframework.cloud>Dalston.SR4</version.org.springframework.cloud>
```

Comment:
Could you please bump the version to Greenwich.RELEASE?

Reply:
I've updated the Spring Cloud version to Greenwich.